fetch.bs

```diff
@@ -7852,7 +7855,7 @@ constructor steps are:
 
   <ol>
    <li><p>Let <var>bodyWithType</var> be the result of <a for=BodyInit>extracting</a>
-   <var>init</var>["{{RequestInit/body}}"], with <a for=BodyInit/extract><var>keepalive</var></a>
+   <var>init</var>["{{RequestInit/body}}"], with <a for=BodyInit/extract>keepalive</a>
```

Comment:
I think this should use `<i>` looking at https://infra.spec.whatwg.org/#example-algorithm-optional-named-params.

Reply:
Ah, good catch